contracts/ExampleNFT-v2.cdc

```diff
@@ -0,0 +1,368 @@
+/* 
+*
+*  This is an example implementation of a Flow Non-Fungible Token
+*  using the V2 standard.
+*  It is not part of the official standard but it assumed to be
+*  similar to how many NFTs would implement the core functionality.
+*
+*  This contract does not implement any sophisticated classification
+*  system for its NFTs. It defines a simple NFT with minimal metadata.
+*   
+*/
+
+import NonFungibleToken from "./NonFungibleToken-v2.cdc"
+import MetadataViews from "./MetadataViews.cdc"
+
+pub contract ExampleNFT: NonFungibleToken {
+
+    /// Standard events from the NonFungibleToken Interface
+
+    pub event Withdraw(id: UInt64, from: Address?, type: Type)
+    pub event Deposit(id: UInt64, to: Address?, type: Type)
+    pub event Transfer(id: UInt64, from: Address?, to: Address?, type: Type)
```

Comment:
Now that NFTs all implement resolveView is it worth it to add some of the views to this event _if_ they are present? Just things like Display and Edition/Serial will give so much value to make an event independent and human readable.

Reply:
I think I'm ok with that. I'll add it to the proposal and we'll see what people think